### PR TITLE
LPD-34975

### PR DESCRIPTION
**Pull request by igor-franca**
Related [ticket](https://liferay.atlassian.net/browse/LPD-26429)

### CODE REVIEW

File: modules/test/playwright/tests/object-web/objectFields.spec.ts [hunk 1]
```diff
@@ -581,10 +587,51 @@ test.describe('Manage objectFields through Objects Admin UI', () => {
 			});
 		}
 
+		while (
+			(await page.locator('.dnd-tbody > .dnd-tr').all()).length !==
+			objectDefinition.objectFields.length + objectFieldsMock.length
+		) {
+			await page.waitForTimeout(1000);
+		}
```
Comment (igor-franca) [inline modules/test/playwright/tests/object-web/objectFields.spec.ts:718]: Hey @danvb112, I'm facing problems when I try to make this change here.

I'm testing locally using the show browser option. `objectFieldTableRows` has fewer expected items using just networkidle.

Comment (danvb112) [reply modules/test/playwright/tests/object-web/objectFields.spec.ts:718]: Ok @igor-franca , so you can keep your original solution :smile:

**Review by danvb112 — APPROVED** [review]
LGTM! Great job @igor-franca :tada:

**Comment by danvb112** [comment]
ci:test:object

**Comment by danvb112** [comment]
ci:test:playwright

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:test:playwright

**Comment by igor-franca** [comment]
ci:test:relevant

**Comment by igor-franca** [comment]
ci:forward:force

**Comment by igor-franca** [comment]
All tests were successful in Jenkins:

https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/359964/testReport/

**Comment by igor-franca** [comment]
ci:forward:force

**Pull request closed without merging**